scripts/release/README.md

```diff
@@ -170,7 +176,9 @@ Upon completion, this script provides instructions for tagging the Git commit th
 **Specify a `--dry` flag when running this script if you want to skip the NPM-publish step.** In this event, the script will print the NPM commands but it will not actually run them.
 
 #### Example usage
-To publish a release to NPM as both `next` and `latest`:
+To publish a release to NPM as `latest`:
 ```sh
-scripts/release/publish.js --tags latest
```

Comment:
This was wrong before. In particular, to publish both `next` and `latest`, you *had to* pass them both. This seems to have gotten lost in an edit (9e158c). Now it actually works.